examples/test.rs

```diff
@@ -47,6 +52,22 @@ fn main() {
                         .save("example_outputs/square_test_result.png")
                         .unwrap()
                 }
+                Format::RG8 => {
+                    let save_image =
+                        GrayAlphaImage::from_vec(target_width, target_height, downsampled_pixels)
```

Comment:
This is a bit ugly 😅

Reply:
What exactly would you like to do differently about it?

Reply:
I don't like that the format says RG8 and the output is RA instead :) But that is the same problem we had when using sbt image.